Makefile

```diff
@@ -596,6 +596,7 @@ test-e2e: test-e2e-setup install-ginkgo ## Run E2E tests against OADP operator i
 	--ginkgo.vv \
 	--ginkgo.no-color=$(OPENSHIFT_CI) \
 	--ginkgo.label-filter="$(TEST_FILTER)" \
+	--ginkgo.junit-report="junit_report.xml" \
```

Comment:
what will this do? write to this file after test run? if so, we need to add file to gitignore

And we also need to check if file is stored in PROW to validate PR? or not needed?

Reply:
I agree, but atm all we would need in this pr is a gitignore.  I have asked for this work to be done so that the QE infra team can get some experience with importing test results into the report portal.